packages/angular/projects/clr-angular/src/utils/popover/providers/popover-toggle.service.spec.ts

```diff
@@ -94,6 +95,30 @@ export default function (): void {
         expect(this.toggleService.open).toBeFalse();
         expect(this.toggleService.openEvent).toBe(closeClickEvent);
       });
+
+      Object.keys(KeyCodes)
+        .filter(key => key.search('Arrow') > -1)
```

Comment:
@hippee-lee @coryrylan I am not really satisfied with getting all the arrow keys based on a hardcoded string in a filter,  but I couldn't think of a better way at this point. If you have suggestions I am open to improving the filter.

Reply:
It works, I'm ok with the approach. 

What do you think about combining into one pass instead of two? e.g 
```typescript
Object.keys(KeyCodes)
  .forEach(key => {
    if(key.search('Arrow') > -1) {
      // Arrow keys are ignored to prevent content from closing the popover content
      it(`prevents the default toggle action for the ${key} key`, function (this: TestContext) {
        const arrowKeyEvent = new KeyboardEvent(key, { key });
        spyOn(arrowKeyEvent, 'preventDefault');
        this.toggleService.toggleWithEvent(arrowKeyEvent);
        expect(arrowKeyEvent.preventDefault).toHaveBeenCalled();
      }
    } else if(key.search('Arrow') === -1) {
      it(`does not prevent the default toggle action for the ${key} key`, function (this: TestContext) {
        const arrowKeyEvent = new KeyboardEvent(key, { key });
        spyOn(arrowKeyEvent, 'preventDefault');
        this.toggleService.toggleWithEvent(arrowKeyEvent);
        expect(arrowKeyEvent.preventDefault).not.toHaveBeenCalled();
      }
    }
  });
```

Reply:
Done.